xml/depl_require.xml

```diff
@@ -1051,10 +1051,10 @@
     </para>
     <para>
      Log files from all nodes in &cloud; are stored on the &admserv;
-     under <filename>/var/log</filename> (see
-     <xref linkend="sec.deploy.logs.adminserv"/> for a complete list).
+     under <filename>/var/log</filename>.
      The message service RabbitMQ requires 1 GB of free space
-     in <filename>/var</filename>.
+     in <filename>/var</filename> (see
+     <xref linkend="sec.deploy.logs.adminserv"/> for a complete list).
```

Comment:
_sec.deploy.logs.adminserv_ is causing some of the build failures

Reply:
I know -- please read paragraph 3 of the description of this PR.

You did not include the log-file chapter file in your merged guide and I was not sure whether that was an oversight or by design.